src/app/demos/scrollspy.component.html

```diff
@@ -9,18 +9,18 @@
     <li class="nav-item">
       <a class="nav-link" (click)="onClick('scrollspyHeading2')">Second</a>
     </li>
-    <li class="nav-item dropdown">
-      <a class="nav-link dropdown-toggle" data-bs-toggle="dropdown" href="#" role="button" aria-expanded="false">Dropdown</a>
-      <ul class="dropdown-menu">
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading3')">Third</a></li>
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading4')">Fourth</a></li>
-        <li><hr class="dropdown-divider"></li>
-        <li><a class="dropdown-item" (click)="onClick('scrollspyHeading5')">Fifth</a></li>
-      </ul>
+    <li ngbDropdown class="nav-item">
+      <a href (click)="false" class="nav-link" ngbDropdownToggle>Dropdown</a>
+      <div ngbDropdownMenu>
+        <button ngbDropdownItem (click)="onClick('scrollspyHeading5')">Third</button>
```

Comment:
`(click)="onClick('scrollspyHeading3')"` (and 4 for the next one)

Reply:
fixed